Readme.md

```diff
@@ -152,6 +152,16 @@ in double quotes, e.g. `qtuicargs { "foo", "bar" }` will appear as `"foo" "bar"`
 An optional list of arguments that will be sent to the Qt rcc tool. Each argument will be encased
 in double quotes, e.g. `qtrccargs { "foo", "bar" }` will appear as `"foo" "bar"` in the command line.
 
+##### qttsargs { "arg", "arg", ... }
+
+An optional list of arguments that will be sent to the Qt lrelease tool. Each argument will be encased
+in double quotes, e.g. `qtuicargs { "foo", "bar" }` will appear as `"foo" "bar"` in the command line.
+
+##### qtqmgenerateddir "path"
+
+The optional path, relative to the current script, where the qm files generated
+by Qt will be created. If omitted, the default behavior is to generate those files near the ts files.
```

Comment:
The default must be to generate either next to the project's executable, or in the same directory as the other generated files. But *never* in the source tree.
If people want to generate stuff in their source trees, I don't mind (this option is perfectly fine for that) but the default should always be to keep the sources clean (e.g. this addon should never force a user to update its `.gitignore` or whatever by default)